app/javascript/modules/listingDetails/ListingDetailsAdditionalInformation.tsx

```diff
@@ -38,7 +38,7 @@ export const ListingDetailsAdditionalInformation = ({
         {(!!listing.Required_Documents || isSale(listing)) && (
           <div className="info-card bg-gray-100 border-0">
             <h3 className="text-serif-lg">{t("listings.requiredDocuments")}</h3>
-            <div className="text-sm">
+            <div className="text-sm translate">
```

Comment:
was expecting more of these based on the compiled list (as noted in the AC). is that coming in a separate pr or separate ticket?

Reply:
I should've made a comment in the ticket! I felt like this was likely to spill over into the next sprint and we're at a good place to make that a new ticket.

Reply:
yes, lets update the AC in the jira ticket so whoever works on testing is up to date. perhaps just cross out the original AC and link to the new ticket